src/stylesheets/button.css

```diff
@@ -0,0 +1,191 @@
+* { outline: 0 !important; }
+
+body {
```

Comment:
Shouldn't be referening `body` or `form` either. Use css classes instead so we don't have to worry about css conflicts with other components and apps